### PR TITLE
Начинаем программировать

### PR DESCRIPTION
**Pull request by olegsorokin**
---
:mortar_board: [Начинаем программировать](https://up.htmlacademy.ru/react/9/user/30818/tasks/3)

### CODE REVIEW

File: project/src/components/main/main.tsx [hunk 1]
```diff
@@ -0,0 +1,193 @@
...
+                points="10 -1.11910481e-13 14 -1.11910481e-13 14 21 10 21"
+              />
+            </g>
+          </symbol>
+        </svg>
+      </div>
+
+      <section className="film-card">
+        <div className="film-card__bg">
+          <img src="img/bg-the-grand-budapest-hotel.jpg" alt={promoFilm.title} />
```
Comment (dianatsevan) [inline project/src/components/main/main.tsx:68]: We could use destructuring for `promoFilm` to not use dot notation here and further

File: project/src/components/main/main.tsx [hunk 2]
```diff
@@ -0,0 +1,193 @@
+import FilmCard from '../film-card/film-card';
+import { Film, Limit } from '../app/app';
+
+type CatalogProps = {
+  limit: Limit,
+  promoFilm: Film
+}
+
+function Main({limit, promoFilm}: CatalogProps): JSX.Element {
```
Comment (dianatsevan) [inline project/src/components/main/main.tsx:9]: What do you think about renaming components of the pages: not just `Main` but `MainPage`. In such case, we would better understand what we see.

File: project/src/components/main/main.tsx [hunk 3]
```diff
@@ -0,0 +1,193 @@
+import FilmCard from '../film-card/film-card';
+import { Film, Limit } from '../app/app';
+
+type CatalogProps = {
+  limit: Limit,
```
[the remark this answers is not in the document]
Comment (olegsorokin) [reply project/src/components/main/main.tsx:5]: Yes! `Limit` type is just a `number`. We can change this type.

File: project/src/components/main/main.tsx (hunk 1 above)
Comment (olegsorokin) [reply project/src/components/main/main.tsx:68]: Like this?
```typescript
...
function Main({
  limit,
  promoFilm: {
    title,
    genre,
    released,
  },
}: Props): JSX.Element {
...
```

File: project/src/components/main/main.tsx (hunk 2 above)
Comment (olegsorokin) [reply project/src/components/main/main.tsx:9]: I think it is some component, not a page, but if the name of component `MainPage` is more declaratory, we will rename it.

**Comment by dianatsevan** [comment]
Good work! Let's go on :)

**Pull request merged**